<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-7IDTCRC" failures="1" tests="2" name="Login Test(failed)" time="4.942" errors="0" timestamp="2022-02-24T20:30:18 EST">
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="getTitleTest" time="0.034">
    <failure type="java.lang.AssertionError" message="expected [Login | Salesforce] but found [Login | Salesforce1]">
      <![CDATA[java.lang.AssertionError: expected [Login | Salesforce] but found [Login | Salesforce1]
at com.SalesForceLoginTests.NegativeLoginTest.getTitleTest(NegativeLoginTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- getTitleTest -->
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="homePage" time="0.0">
    <skipped/>
  </testcase> <!-- homePage -->
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="passwordButton" time="0.0">
    <ignored/>
  </testcase> <!-- passwordButton -->
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="userNameButton" time="0.0">
    <ignored/>
  </testcase> <!-- userNameButton -->
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="signinButton" time="0.0">
    <ignored/>
  </testcase> <!-- signinButton -->
  <testcase classname="com.SalesForceLoginTests.NegativeLoginTest" name="loginTest" time="0.0">
    <ignored/>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Login Test(failed) -->
